VocaDbModel/Service/VideoServices/VideoServiceBilibili.cs

```diff
@@ -105,6 +113,7 @@ public class BiliMetadata
 
 class BilibiliResponse
 {
+	public int Code { get; init; } = default!;
 	public BilibiliResponseData Data { get; init; } = default!;

```

Comment:
The `default!` expression means, that Code and Data can't be null, but in your error case Data should be null (because it isn't present in the json response). Could you change Data to be nullable and check if this causes errors somewhere else in the code?

Reply:
Thank you for pointing that out! With zero C# experience, I am frankly not very sure how to do that. From what I can tell, it would be `BilibiliResponseData? Data { get; init; } = default;`? Is that right?

Reply:
Yes, that is correct